python/pyspark/sql/readwriter.py

```diff
@@ -97,7 +97,8 @@ def load(self, path=None, format=None, schema=None, **options):
         :param schema: optional :class:`StructType` for the input schema.
         :param options: all other string options
 
-        >>> df = sqlContext.read.load('python/test_support/sql/parquet_partitioned')
+        >>> df = sqlContext.read.load('python/test_support/sql/parquet_partitioned', opt1 = True,
+        ...     opt2 = 1, opt3 = 'str')
```

Comment:
to be more explicit, the style guide in Python is to not put spaces around = for named parameters.